go/enclave/bridge/bridge.go

```diff
@@ -163,7 +163,7 @@ func (bridge *Bridge) ExtractRollups(b *types.Block, blockResolver db.BlockResol
 			// In case of L1 reorgs, rollups may end published on a fork
 			if blockResolver.IsBlockAncestor(b, r.Header.L1Proof) {
 				rollups = append(rollups, obscurocore.ToEnclaveRollup(r, bridge.TransactionBlobCrypto))
-				common.LogWithID(bridge.NodeID, "Extracted Rollup r_%d from block b_%d",
+				common.TraceWithID(bridge.NodeID, "Extracted Rollup r_%d from block b_%d",
```

Comment:
Do we have trace level logging in DataDog atm? If not we should probably check with Tudor that it isn't one of his go-to debugging log msgs.

Reply:
Good point, checked with Tudor, looks good 👍